src/DigitalSubscriberAppBanner/index.stories.tsx

```diff
@@ -1,8 +1,8 @@
 import React, { ReactElement } from 'react';
 import { withKnobs, text } from '@storybook/addon-knobs';
-import { BrazeMessage } from './BrazeMessage';
-import { StorybookWrapper } from './utils/StorybookWrapper';
-import { knobsData } from './utils/knobsData';
+import { BrazeMessage } from '../BrazeMessage';
```

Comment:
Reminder to ourselves that we talked about ways to avoid having to update relative paths when we move files around. I used https://github.com/tleunen/babel-plugin-module-resolver on a previous project which allows you to define aliases/named entry points and it worked well, so that could be an option (not sure how well it plays with TypeScript, we'd need to try it out).